core/templates/dev/head/pages/exploration_editor/translation_tab/audio_translation_bar_directive.html

```diff
@@ -1,6 +1,7 @@
 <div  class="oppia-drop-area" ng-if="showDropArea">
   <div class="oppia-blur-background"></div>
-  <div class="oppia-drop-area-message">Drop your file here!</div>
+  <div class="oppia-drop-area-message" ng-if="guest == false && disallowed == false">Drop your file here!</div>
```

Comment:
Add a translation for the message.

Reply:
I did have one but in this [doc](https://docs.google.com/document/d/1vX0ik2fAbgyKC29QXPBVtgvV-sb47DLNPUf-FmPtAM4/edit#heading=h.jlyga2u0qfrc) I was told to remove it.

Reply:
I think you might both be referring to different things here...

- We talked about removing the message for the case where the user is logged in but has not enough rights. I think that still makes sense (and I left a further comment in @mzaman07's doc). 
- When @bansalnitish is talking about translations I think he means replacing the text with an I18N code. However, I want to point out that this is not necessary here because we don't currently translate text in the editor tab.

Reply:
I see

Reply:
The confusion has been cleared up so it's done.